src/plugins/mer/mervmselectiondialog.cpp

```diff
@@ -48,6 +48,8 @@ MerVmSelectionDialog::MerVmSelectionDialog(QWidget *parent)
     execAsynchronous(std::tie(unusedVms, ok), Sdk::unusedVirtualMachines);
     QTC_CHECK(ok);
     for (const VirtualMachineDescriptor &vm : unusedVms) {
+        if (vm.name == "<inaccessible>")
+            continue;
```

Comment:
It should be fixed in `Sfdk::VBoxVirtualMachinePrivate::listedVirtualMachines` instead.

Reply:
Ok, done.